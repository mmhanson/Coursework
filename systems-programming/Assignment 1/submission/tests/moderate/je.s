main:
	movl	$0, %eax
	movl	$0, %ebx
	movl	$1, %ecx
	cmpl	%eax, %ebx
	je	pass1
	movl	$15, %edi
	printr	%edi
pass1:
	movl	$10, %edi
	printr	%edi
	cmpl	%eax, %ecx
	je	fail2
	movl	$11, %edi
	printr	%edi
fail2:
	ret
