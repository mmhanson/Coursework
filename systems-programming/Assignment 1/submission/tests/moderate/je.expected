10 (0xa)
11 (0xb)
